inductiva/storage/storage.py

```diff
@@ -0,0 +1,89 @@
+"""Methods to interact with the user storage resources."""
+import inductiva
+from inductiva.client.apis.tags import instance_api
+from inductiva.utils import format_utils
+from typing import Literal
+
+
+def get_space_used():
+    """Returns the occupied storage size in GB."""
+    try:
+        api = instance_api.InstanceApi(inductiva.api.get_client())
+        response = api.get_storage_size()
+        storage_used = round(float(response.body), 3)
+        print(f"Total storage used: {storage_used} GB")
+        return storage_used
+    except inductiva.client.ApiException as api_exception:
+        raise api_exception
+
+
+def listdir(max_results: int = 3,
```

Comment:
I feel that part of the reason why naming this method was a bit challenging (and now I understand some of the points raised by @luispcunha wrt to naming it a listdir) is that this method only allows listing the top dir of the user. I think that is a bit of unnatural and unexpected from a UX point of view. Why can't I also list the contents of one of the sub-buckets? Again, we need to make the interaction as similar as possible to what users can in regular cloud / local systems. People can list the contents of the directory and then they can list their subdirectories.

I understand that there are some challenges here in providing the ability to navigate the user's bucket but, with a good design, this can be made secure. For me, the expected prototype of the "list/listdir" method should be:

```
def listdir(path --> default = "/" i.e. top directory
                  max_results: int = 3,
                  order_by: Literal["size", "creation_time"] = "size",
                  sort_order: Literal["asc", "desc"] = "desc")
```

so that if you don't add path, then you get the behaviour you have now, but if you want to list the content of directory "123" you can do:
```
p = inductiva.storarge.listdir("123")
```

Reply:
In our current infrastructure, user buckets are organized with folders corresponding to each task. These task folders contain two files: `output.zip` and `input.zip`. Given this structure, listing the contents of each task folder becomes redundant (it will show the same information for every task directory).  To see the contents of the output.zip inside the task folder users can use the method `task.get_output_files_info()`. The purpose of the feature I am introducing in this PR would be a workflow like this: 
**Usage Example:**
```python
# Check storage for task directories
inductiva.storage.listdir()

# Output:
#   Name       Size        Creation Time
#   1234       5.68 MiB    29 Sep, 14:12:00
#   12345      374.85 KiB  29 Sep, 14:13:10
#   1234567    97.59 KiB   29 Sep, 14:13:24
#   123        0 B         29 Sep, 14:13:29
#   123456     0 B         29 Sep, 14:13:18

# Inspect the contents of the largest task directory (e.g., '1234')
inductiva.Task("1234").get_output_files_info()

# Delete a specific task directory (e.g., '123')
inductiva.storage.delete_task_storage(task_id="123")
```